### PR TITLE
added unit test for function in mapper-thrift-configstore file

### PR DESCRIPTION
**Pull request by d-vignesh**
**What changed?**
Added unit tests for functions under mapper/thrift/configStore.go file

**Why?**
To improve test coverage.

**How did you test it?**
Locally using `go test`.

**Potential risks**

**Release notes**

**Documentation Changes**

### CODE REVIEW

File: common/types/mapper/thrift/configStore_test.go [hunk 1]
```diff
@@ -0,0 +1,156 @@
+package thrift
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
```
Comment (taylanisikdemir) [inline common/types/mapper/thrift/configStore_test.go:6]: files need the license thing at the top and imports follow certain ordering. if you run `make fmt` it should take care of these.

**Review by taylanisikdemir — COMMENTED** [review]
Thanks for contributing to coverage improvement!

File: common/types/mapper/thrift/configStore_test.go (hunk 1 above)
Comment (d-vignesh) [reply common/types/mapper/thrift/configStore_test.go:26]: updated the file. Can you please check once.

Comment (3vilhamster) [reply common/types/mapper/thrift/configStore_test.go:26]: You'll also need to run
make fmt.
We use gci to ensure imports sort/grouping/order.

Comment (d-vignesh) [reply common/types/mapper/thrift/configStore_test.go:26]: yeah added the license info and ran make fmt command also.

File: common/types/mapper/thrift/configStore_test.go [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+func TestDynamicConfigBlob(t *testing.T) {
+	testCases := []struct {
+		desc  string
+		input *types.DynamicConfigBlob
+	}{
+		{
+			desc:  "non-nil input test",
+			input: &testdata.DynamicConfigBlob,
+		},
+		{
```
Comment (Shaddoll) [inline common/types/mapper/thrift/configStore_test.go:41]: Could you add a new case `empty struct`?
And do you mind renaming the file to snake_case format?

Comment (d-vignesh) [reply common/types/mapper/thrift/configStore_test.go]: changed file name and added additional test cases @Shaddoll

**Comment by d-vignesh** [comment]
Hi team, the CI pipeline seems to be failing with the last push when i tried rebasing with the master, not able to get any error logs from the pipeline, can you please check what is the issue

**Review by Shaddoll — APPROVED** [review]

**Pull request merged**